docs/options.md

```diff
@@ -70,7 +70,7 @@ Here are all the available options to call with `train.py`
 | --G_unet_mha_num_heads | int | 1 | number of heads in the mha architecture |
 | --G_unet_mha_res_blocks | array | [2, 2, 2, 2] | distribution of resnet blocks across the UNet stages, should have same size as --G_unet_mha_channel_mults |
 | --G_unet_mha_vit_efficient | flag |  | if true, use efficient attention in UNet and UViT |
-| --G_unet_vid_max_frame | int | 24 | max frame number for unet_vid in the PositionalEncoding |
+| --vid_max_sequence_length | int | 25 | max frame number for unet_vid in the PositionalEncoding |
```

Comment:
Rename `--G_unet_vid_max_sequence_length`  since this is a property of the unet_vid architecture, not the data or the video.

Reply:
OK